qiita_db/metadata_template/util.py

```diff
@@ -337,6 +331,36 @@ def get_invalid_sample_names(sample_names):
     return inv
 
 
+def get_invalid_column_names(column_names):
```

Comment:
given the import, my guess is that's only safe if py3. but i see where letters is coming from now so i htink this is fine